src/gluonnlp/data/transforms.py

```diff
@@ -988,10 +1004,261 @@ def convert_tokens_to_ids(self, tokens):
+          sequence or the second sequence.
+        - generate valid length
+
+        For sequence pairs, the input is a tuple of 3 strings:
+        text_a, text_b and label.
+
+        Inputs:
+            text_a: 'is this jacksonville ?'
+            text_b: 'no it is not'
+            label: '0'
+        Tokenization:
+            text_a: 'is this jack ##son ##ville ?'
+            text_b: 'no it is not .'
+        Processed:
+            tokens:  '[CLS] is this jack ##son ##ville ? [SEP] no it is not . [SEP]'
+            type_ids: 0     0  0    0    0     0       0 0     1  1  1  1   1 1
+            valid_length: 14
+            label: 0
+
+        For single sequences, the input is a tuple of 2 strings: text_a and label.
+        Inputs:
+            text_a: 'the dog is hairy .'
+            label: '1'
+        Tokenization:
+            text_a: 'the dog is hairy .'
+        Processed:
+            text_a:  '[CLS] the dog is hairy . [SEP]'
+            type_ids: 0     0   0   0  0     0 0
+            valid_length: 7
+            label: 1
+
+        Parameters
+        ----------
+        line: tuple of str
+            Input strings. For sequence pairs, the input is a tuple of 3 strings:
+            (text_a, text_b, label). For single sequences, the input is a tuple
+            of 2 strings: (text_a, label).
+
+        Returns
+        -------
+        np.array: input token ids in 'int32', shape (batch_size, seq_length)
+        np.array: valid length in 'int32', shape (batch_size,)
+        np.array: input token type ids in 'int32', shape (batch_size, seq_length)
+        np.array: label id in 'int32', shape (batch_size, 1)
+        """
+        label = line[-1]
+        label = convert_to_unicode(label)
+        label_id = self._label_map[label]
+        label_id = np.array([label_id], dtype='int32')
+        input_ids, valid_length, segment_ids = self._bert_xform(line[:-1])
+        return input_ids, valid_length, segment_ids, label_id
+
+
 def _whitespace_tokenize(text):
     """Runs basic whitespace cleaning and splitting on a piece of text."""
     text = text.strip()
     if not text:
         return []
     tokens = text.split()
     return tokens
+
+
+def convert_to_unicode(text):
+    """Converts `text` to Unicode (if it's not already), assuming utf-8 input."""
+    if six.PY3:
+        if isinstance(text, str):
+            return text
+        elif isinstance(text, bytes):
+            return text.decode('utf-8', 'ignore')
+        else:
+            raise ValueError('Unsupported string type: %s' % (type(text)))
+    elif six.PY2:
+        if isinstance(text, str):
+            return text.decode('utf-8', 'ignore')
+        elif isinstance(text, unicode):  # noqa: F821
+            return text
+        else:
+            raise ValueError('Unsupported string type: %s' % (type(text)))
+    else:
+        raise ValueError('Not running on Python2 or Python 3?')
+
+
+def _truncate_seq_pair(tokens_a, tokens_b, max_length):
```

Comment:
This is specific for BERTClassificationTransform. Put it in that class

Reply:
ok.